search/test.py

```diff
@@ -0,0 +1,56 @@
+import json
+from unittest import TestCase
+
+from search.views import *
+from user.models import User
+from core import app
+
+
+class SearchTest(TestCase):
+    def setUp(self):
+        self.client = app.test_client()
+
+    def tearDown(self):
```

Comment:
This copy paste may mean it's time to merge some code.
You may want to create something similar to `core.Testcase` and extend it.